susemanager-utils/testing/automation/schema-migration-test-pgsql.sh

```diff
@@ -4,6 +4,23 @@ HERE=`dirname $0`
 . $HERE/VERSION
 GITROOT=`readlink -f $HERE/../../../`
 
+# File created by Gitarro with info about a PR, it only exists when we are testing a PR
+GITARRO_JSON="${GITROOT}/.gitarro_pr.json"
+
+# Base version for the idempotency test. All PostgreSQL since this version must be idempotent for the test to pass
+# This value is is used if the file for the PR is not found (we assume we are on a branch)
+BASE_VERSION='4.0.1'
+
 # we need a special (old) baseimage to migrate to current schema
 docker pull $REGISTRY/$PGSQL_CONTAINER
-docker run --privileged --rm=true -v "$GITROOT:/manager" $REGISTRY/$PGSQL_CONTAINER /manager/susemanager-utils/testing/docker/scripts/schema_migration_test_pgsql-31to40.sh
+
+# Check if we run for PR or not
+if [ -f ${GITARRO_JSON} ]; then
+  echo "Running from PR"
+  IDEMPOTENCY_PARAMS=" -p ${GITARRO_JSON}"
+else 
+  echo "Running from Branch"
+  IDEMPOTENCY_PARAMS=" -v ${BASE_VERSION}"  
+fi
+
+docker run --privileged --rm=true -v "$GITROOT:/manager" $REGISTRY/$PGSQL_CONTAINER /bin/bash -c "/manager/susemanager-utils/testing/docker/scripts/schema_migration_test_pgsql-31to40.sh && /manager/susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py ${IDEMPOTENCY_PARAMS}"
```

Comment:
we could grap the Scripts in variable
like this one
`/manager/susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py`

I think this var can stay in this script since is local to this one

Reply:
Fixed and testing.